eng/Versions.props

```diff
@@ -4,7 +4,7 @@
     <MajorVersion>6</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <PatchVersion>22</PatchVersion>
```

Comment:
```suggestion
    <PatchVersion>23</PatchVersion>
```

Reply:
We should probably construct the full version from the parts now that both exist